src/bilder/images/edxapp/deploy.py

```diff
@@ -187,14 +248,28 @@
     service_configuration_watches(
         service_name="edxapp-cms",
         watched_files=[studio_config_path],
+        start_now=False,
         onchange_command=(
             # Let edxapp read the rendered config file
             f"/bin/bash -c 'chown edxapp:www-data {studio_config_path} &&"  # noqa: WPS237, WPS221, E501
             # Ensure that Vault can update the file when credentials refresh
-            f" setfacl -m u:vault:rwx {studio_config_path} &&"
+            f" setfacl -m u:consul-template:rwx {studio_config_path} &&"
             # Restart the edxapp process to reload the configuration file
             " /edx/bin/supervisorctl restart "
             f"{'cms' if node_type == WEB_NODE_TYPE else 'all'}'"
         ),
     )
+    service_configuration_watches(
+        service_name="edxapp-forum",
+        watched_files=[forum_config_path],
+        start_now=False,
+        onchange_command=(
+            # Let edxapp read the rendered config file
+            f"/bin/bash -c 'chown forum:www-data {forum_config_path} &&"  # noqa: WPS237, WPS221, E501
+            # Ensure that Vault can update the file when credentials refresh
+            f" setfacl -m u:consul-template:rwx {forum_config_path} &&"
+            # Restart the edxapp process to reload the configuration file
+            " /edx/bin/supervisorctl restart forum"
```

Comment:
nit: extra space

Reply:
These are actually intentional spaces because it's concatenated to the previous line. I could add the space as a trailing value to the previous line. Probably worth deciding what the coding standard is for that. I don't have any strong opinion.